File: Aula - Março 18/principal.java
import java.util.ArrayList;

public class Principal {
    public static void main(String[] args) {
        ArrayList<Clima> lista = new ArrayList<>();
        /*Clima umClima = new Clima (ano: "2020", mes: "Janeiro", temperatura: "Quente", precipitacao: "Pouca")*/

        String linha = "2020,Janeiro,Quente,Pouca";
        String dadosLinha[] = linha.split(","); 
        Clima objClima = new Clima (dadosLinha[0], dadosLinha[1], dadosLinha[2], dadosLinha[3]);
        if (!lista.contains(objClima)) {
            lista.add (objClima);
        }

        //segunda linha
        linha = "2020,Janeiro,Frio,Muita";
        dadosLinha = linha.split(",");
        objClima = new Clima(dadosLinha[0], dadosLinha[1], dadosLinha[2], dadosLinha[3]);
        if (!lista.contains(objClima)) {
            lista.add (objClima);
    }

        for (Clima c : lista) {
            System.out.println(c);
        }
    }
}

File: Aula - Março 18/principal.py
from clima import Clima

lista = []

linha = "2020, Janeiro, Quente, Pouca";
dadosLinha = linha.split(",");

    
obj_Clima = Clima (dadosLinha[0], dadosLinha[1], dadosLinha[2], dadosLinha[3]);

if obj_Clima not in lista:
    lista.append(obj_Clima)

#sugunda linha
linha = "2020, Janeiro, Frio, Muita";
dadosLinha = linha.split(",");

    
obj_Clima = Clima (dadosLinha[0], dadosLinha[1], dadosLinha[2], dadosLinha[3]);

if obj_Clima not in lista:
    lista.append(obj_Clima)

for c in lista:
    print(c)
